catalogd/config/components/ca/patches/manager_deployment_cacerts.yaml

```diff
@@ -6,4 +6,4 @@
   value: {"name":"olmv1-certificate", "readOnly": true, "mountPath":"/var/ca-certs/"}
 - op: add
   path: /spec/template/spec/containers/0/args/-
-  value: "--ca-certs-dir=/var/ca-certs"
+  value: "--pull-cas-dir=/var/ca-certs"
```

Comment:
I know this is upstream and doesn't affect our downstream runs, but this seems incorrect.  Won't this mean that catalogd can only talk to image registries whose certs are signed by the olmv1-certificate CA?

Reply:
Only if we specify `--pull-cas-dir` in the catalogd manifests, which we don't (or shouldn't).

Reply:
This is _in addition_ to the system cert pool, so even upstream, the it'll use the system pool certs for anything not found here. If this is set, it won't use /etc/docker/certs.d, though.

Reply:
> specify --pull-cas-dir in the catalogd manifests

Isn't that what this line is doing?

Reply:
Nevermind! You righted me over in the other thread about this. I keep forgetting the defaulting nuances of `containers/image`.

Reply:
But this is also only for upstream, if you look at the downstream manifests, the flag is not used.